python/templates/CollectionData.h.jinja2

```diff
@@ -83,11 +84,11 @@ public:
 private:
   // members to handle 1-to-N-relations
 {% for relation in OneToManyRelations %}
-  podio::UVecPtr<{{ relation.namespace }}::{{ relation.bare_type }}> m_rel_{{ relation.name }};  ///< Relation buffer for read / write
+  podio::UVecPtr<{{ relation.namespace }}::{{ relation.bare_type }}> m_rel_{{ relation.name }}{new std::vector<{{ relation.namespace }}::{{ relation.bare_type }}>()};  ///< Relation buffer for read / write
```

Comment:
Not sure if it's more readable as it is already quite bad at the moment, but could we do this with `std::make_unique`?